db/tables.py

```diff
@@ -44,15 +44,13 @@ def reflect_table(name, schema, engine):
     return Table(name, metadata, schema=schema, autoload_with=engine)
 
 
-def reflect_table_columns(name, schema, engine):
-    t = reflect_table(name, schema, engine)
-    return [
-        {"name": c.name, "type": c.type} for c in t.columns
-    ]
+def get_records(table, engine, limit=None, offset=None):
+    query = select(table).limit(limit).offset(offset)
+    with engine.begin() as conn:
+        return conn.execute(query).fetchall()
 
 
-def get_all_table_records(name, schema, engine):
-    t = reflect_table(name, schema, engine)
-    sel = select(t)
+def get_count(table, engine):
+    query = select([func.count()]).select_from(table)
```

Comment:
As noted elsewhere, this will be extremely slow (think dozens of minutes or more) for tables with a large number of rows.  We should use it judiciously.